README.md

```diff
@@ -508,7 +508,7 @@ $
 To wipe everything clean:
 
 ```bash
-rm -rfi ~/.binenv ~/.config/binenv/
```

Comment:
This is intentional so the use has to confirm removal explicitely. Better be on the safe side :D

Reply:
but why then the -f? that's kind of contradictory, but okay, i will adapt it :D

Reply:
> but why then the -f? that's kind of contradictory, but okay, i will adapt it :D

Agreed that in the context, it does not make sense. I will remove it.